docs/relativecdk.rst

```diff
@@ -0,0 +1,30 @@
+.. _relativeCDK2:
```

Comment:
For the examples, it's probably better to keep the documentation in the `README.md` file in each example directory.

Reply:
Sorry that wasn't supposed to be committed